src/types.ts

```diff
@@ -94,6 +94,16 @@ export type Config = {
   sequelize?: any;
   eventListeners?: EventListener[];
   easterEgg?: any;
+  mergeSettings?: MergeSettings;
+};
+
+export type MergeSettings = {
+  /** Skip the destination branch build when there are successful dependent requests awaiting merge.
+   * This prevents multiple branch builds triggering multiple merges happen in quick succession.
+   * Achieved by adding [skip ci] to the merge commit message
```

Comment:
What consumes the [skip ci] commit message?

Reply:
Hmm that is interesting, but not a concern here since it affects only the build that runs on the target branch after merging the PR.

Reply:
LGTM :)